src/analysis/normed_space/multilinear.lean

```diff
@@ -874,7 +874,41 @@ linear_map.mk_continuous₂
     (λ f g₁ g₂, by { ext1, apply f.map_add }) (λ c f g, by { ext1, simp }))
   1 $ λ f g, by { rw one_mul, exact f.norm_comp_continuous_multilinear_map_le g }
 
-variables {𝕜 E G G'}
+variables {𝕜 G G'}
+
+/-- `continuous_linear_map.comp_continuous_multilinear_map` as a bundled
+continuous linear equiv. -/
+def _root_.continuous_linear_equiv.comp_continuous_multilinear_mapL (g : G ≃L[𝕜] G') :
```

Comment:
Can you add the lemma that `(continuous_linear_equiv g).symm = continuous_linear_equiv g.symm`?